frontend/components/branchViewLink.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { SITE } from '../propTypes';
+import appConfig from '../../config/app';
```

Comment:
Yeah, I kinda agree with feeling a littler nervous about pulling in config to the front end. I saw it was already done in the `AddSite/index.js` component to get the `templates` config.

hrmmmmmmmm....